stackage/ghc-9.2.4/cabal.config

```diff
@@ -0,0 +1,200 @@
+constraints:
```

Comment:
As above, I would move `cabal.config` to sit beside the `stack.yaml` in the repository root.

Reply:
Done.